src/views/PlanTargets.vue

```diff
@@ -137,9 +147,10 @@ export default {
       for (var i = 0; i < this.easylist.length; ++i) {
         var altstart = helpers.eq2altazWithDate(this.easylist[i].ra, this.easylist[i].dec, this.lat1, this.lon1, starttime)[0]
         var altend = helpers.eq2altazWithDate(this.easylist[i].ra, this.easylist[i].dec, this.lat1, this.lon1, endtime)[0]
-        if (altstart>35 && altend>35) {
+        if (altstart>45 && altend>45) {
```

Comment:
If the decision was made based off of airmass, then maybe include a comment saying we picked 45 degrees to satisfy 1.6 airmass or something like that.